network-api/networkapi/buyersguide/utils.py

```diff
@@ -11,24 +11,9 @@ def get_category_og_image_upload_path(instance, filename):
     )
 
 
-def tri_to_quad(input):
-    if input is True:
-        return 'Yes'
-    if input is False:
-        return 'No'
-    return 'U'
-
-
-def quad_to_tri(input):
-    if input == 'Yes':
-        return True
-    if input == 'No':
-        return False
-    return None
```

Comment:
I couldnt find any instances of these being used anymore. So they were removed.

Reply:
they used to be part of the migrations, but it looks like the previous squash already obviated their need.